katalog/external-dns/base/deploy.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      serviceAccountName: external-dns
+      containers:
+      - name: external-dns
+        image: k8s.gcr.io/external-dns/external-dns:v0.10.2
```

Comment:
I think we should move this image to our registry

Reply:
You are absolutely right. Fixed in the last commit ([69a9ae8](https://github.com/sighupio/fury-kubernetes-ingress/pull/81/commits/69a9ae88049c656816365ab531ad59a29f1203bb)) with a Kustomize patch